cogs/marriage.py

```diff
@@ -236,6 +236,41 @@ def reactioncheck(reaction, user):
         await user.send(
             f"**{ctx.author}** bought you a **{item[1]}** and increased your love score by **{item[2]}** points!"
         )
+        
+    @has_char()
+    @commands.command(name="date")
+    @user_cooldown(43200)
+    async def _date(self, ctx):
+        """Take your loved one on a date to increase your lovescore."""
+        num = random.randint(1, 15) * 10
+        async with self.bot.pool.acquire() as conn:
+            marriage = await conn.fetchval(
```

Comment:
`ctx.character_data["marriage"]` already exists after the `has_char`, no need to fetch it from the database once more. That's the last thing before merging, I guess

Reply:
alright, changed